code/game/turfs/turf_ao.dm

```diff
@@ -29,6 +29,8 @@
 	var/image/I = image('icons/turf/flooring/shadows.dmi', cstr, dir = 1 << (i-1))
 	I.alpha = WALL_AO_ALPHA
 	I.blend_mode = BLEND_OVERLAY
+	I.plane = HIDING_MOB_PLANE
```

Comment:
Shouldn't this be `ABOVE_TURF_PLANE`?

Reply:
You've got me!
I wanted it above the shallow fluid layer, but that's above the hiding mobs, so this will make AO render above those, which I'm not sure is reasonable, and it also makes its position in the ordering a little confusing (given you would indeed expect it to be "above turf", so I'll probably shift it to above turf (layer 12)